### ISSUE TITLE
Issue with appengine-python-falcon

### ISSUE THREAD

**Issue by iambillmccann** [GoogleCloudPlatform/community#392]
Issue Description

I ran through the entire tutorial for Falcon on App Engine. Step 2. in the "Running the app" section says:

> Execute the following command to run the app.

dev_appserver.py

> 

When executing the command, I received the error message:

> dev_appserver.py: error: too few arguments

> 

I scanned the arguments listed with the error, and discovered the command requires the path to app.yaml, so I reran the command like this:

> dev_appserver.py "./"

> 

This starts the server. But when I hit the server with a request, I receive an http 500 error. The console shows the following stack trace and message:

> ERROR    2018-03-20 15:47:19,345 wsgi.py:263]
Traceback (most recent call last):
  File "C:\Program Files (x86)\Google\Cloud SDK\google-cloud-sdk\platform\google_appengine\google\appengine\runtime\wsgi.py", line 240, in Handle
    handler = _config_handle.add_wsgi_middleware(self._LoadHandler())
  File "C:\Program Files (x86)\Google\Cloud SDK\google-cloud-sdk\platform\google_appengine\google\appengine\runtime\wsgi.py", line 299, in _LoadHandler
    handler, path, err = LoadObject(self._handler)
  File "C:\Program Files (x86)\Google\Cloud SDK\google-cloud-sdk\platform\google_appengine\google\appengine\runtime\wsgi.py", line 85, in LoadObject
    obj = __import__(path[0])
ImportError: No module named api
INFO     2018-03-20 11:47:19,371 module.py:835] default: "GET / HTTP/1.1" 500 -

>

**Comment by ToddKopriva** [comment]
@archelogos , this issue is about a tutorial that you created. Would you have a look?

**Comment by archelogos** [comment]
Hi there @ToddKopriva @iambillmccann! Happy to see that you found this tutorial useful! 

The dev_appserver.py always needs a parameter, I admit it's not easy to see but there is a dot in the documentation after that command, pointing to the current path (that is where the app.yaml is saved).

Also, regarding the second issue. Make sure you have the provided code downloaded on the same path and within a folder called "api". This is essential to make it work since the app.yaml looks for an "api" module where an "app" variable is defined.

To sum up:
```
- app.yaml
- requirements.txt
- appengine_config.py
- api
  - __init__.py
  - resources.py
  - hooks.py
  - middleware.py
```

and `dev_appserver.py .`
